share/raja/cmake/RAJA-config.cmake.in

```diff
@@ -30,21 +30,13 @@
 #
 # That's all!
 #
-if (NOT RAJA_CONFIG_LOADED)
-  set(RAJA_CONFIG_LOADED TRUE)
+@PACKAGE_INIT@
 
-  if (NOT DEFINED camp_DIR)
-    if (EXISTS @EXTERNAL_CAMP_INSTALL_DIR@)
-      set(camp_DIR @EXTERNAL_CAMP_INSTALL_DIR@)
-    else ()
-      set(camp_DIR @CMAKE_INSTALL_PREFIX@/lib/cmake/camp)
-    endif ()
-  endif ()
-  find_package(camp REQUIRED PATHS ${camp_DIR} NO_DEFAULT_PATH)
-  include(@CMAKE_INSTALL_PREFIX@/share/raja/cmake/RAJA.cmake)
-endif()
+if (NOT TARGET camp)
+  include(CMakeFindDependencyMacro)
+  find_dependency(camp REQUIRED PATHS "@PACKAGE_CMAKE_INSTALL_PREFIX@")
+endif ()
 
-# Export version number
-set(RAJA_VERSION_MAJOR @RAJA_VERSION_MAJOR@)
```

Comment:
I removed the version stuff here and used `write_basic_package_version_file`

Reply:
Will these macros still get set? https://github.com/LLNL/RAJA/blob/develop/include/RAJA/config.hpp.in#L274

Projects use these to check our version information.

Reply:
I think these now come in from the generated RAJA.cmake don't they?

Reply:
Yup, the macros are still set - they get generated at config time. This change lets a project put the version number in the `find_package(RAJA)` statement in their own CMake, if they care to do that.

Reply:
@trws @rhornung67 - I think that these variables will actually be set, based on the following, from https://cmake.org/cmake/help/latest/command/find_package.html#config-mode-version-selection:

```
These variables are checked by the find_package command to determine whether the configuration file provides an acceptable version. They are not available after the find_package call returns. If the version is acceptable the following variables are set:

<PackageName>_VERSION

    Full provided version string
<PackageName>_VERSION_MAJOR

    Major version if provided, else 0
<PackageName>_VERSION_MINOR

    Minor version if provided, else 0
<PackageName>_VERSION_PATCH

    Patch version if provided, else 0
```